tests/Unit/PackagePrivate/Humanizer/Internationalization/ArrayMessageBuilderTest.php

```diff
@@ -39,13 +39,42 @@ public function testParameters(): void {
 		$builder = new ArrayMessageBuilder(
 			[
 				'right-key' => 'This was written by $2 on $1 ($1)',
+				"edtf-day-and-year" => "$1e{{PLURAL:$1|r|}} jour d’un mois inconnu de $2",
+				"month-number" => "... and $1 month{{PLURAL:$1||s}}",
+				"month-number-missing-argument" => "$1 and $2 month{{PLURAL:$2||s}}",
```

Comment:
Tests should test only one thing. At the very least, your new code needs to go into a `testPlural` method. Though more typically you'd create one method for each of the new cases you added. Here are the to me most obvious cases to test:

* first element gets selected
* second element gets selected
* element that is empty gets selected
* parameter is missing

Reply:
Though TBH I'm not sure at the moment if that example method name is correct. Does the code return the first element or an empty string? The production code is not trivial to read.